src/System.Net.Security/ref/System.Net.Security.cs

```diff
@@ -133,6 +133,7 @@ public partial class SslStream : AuthenticatedStream
         public override int Read(byte[] buffer, int offset, int count) { return default(int); }
         public override long Seek(long offset, System.IO.SeekOrigin origin) { return default(long); }
         public override void SetLength(long value) { }
+        public virtual System.Threading.Tasks.Task ShutdownAsync() { return default(System.Threading.Tasks.Task); }
```

Comment:
This API needs to be added as .NET Core specific instead of as being part of netstandard. See https://github.com/dotnet/corefx/pull/12311 as an example.